### ISSUE TITLE
invalidHandler doesn't fire on first submit in some cases.

### ISSUE THREAD

**Issue by davebednarski** [jquery-validation/jquery-validation#1811]
### Subject of the issue

invalidHandler only works after second form submit and subsequent submits, but not on the first submit.  This is when using .valid() method and with a specific HTML markup (see note below)
### Your environment
- version of `jquery-validate` = 1.15.0
- which browser and its version = Chrome (mac) latest version
### Steps to reproduce
1.  Fill in password input field with some value.
2.  Fill in second password re-enter field
3.  click submit
   https://jsfiddle.net/zLe8x5bt/8/
### Expected behaviour

The invalidHandler should fire.
### Actual behaviour

The invalidHandler doesn't fire.  But will fire the second time submit is clicked and subsequent times.
### Notes

Oddly it seems if you wrap the inputs fields in a div the issue goes away.  But if you take the div wrappers out then it comes back.
https://jsfiddle.net/zLe8x5bt/6/

**Comment by sparky672** [comment]
There is no bug.  You're simply misinterpreting what's happening here.

When you attempt to click the submit button, you're actually only triggering validation on the second field due to the focus-out event.  The error message pushes the button out of the way before the mouse-up event completes the click.  When you disable focus-out, you can see it "appear" to behave normally because there is no error message before the button's mouse-up event.  

[https://jsfiddle.net/zLe8x5bt/12/](https://jsfiddle.net/zLe8x5bt/12/)

In other words, you don't actually click the button in your original "bug" scenario.

> Oddly it seems if you wrap the inputs fields in a div the issue goes away.

When you wrap everything in a `div`, it places the button where it does not get pushed out from under your mouse by an error message.
